Sources/AudioKitUI/Visualizations/FFTView.swift

```diff
@@ -25,46 +26,37 @@ class FFTModel: ObservableObject {
         }
     }
 
-    func updateAmplitudes(_ fftFloats: [Float]) {
-        var fftData = fftFloats
-        for index in 0 ..< fftData.count {
-            if fftData[index].isNaN { fftData[index] = 0.0 }
```

Comment:
I didn't see the point of this since we later on ensure that all values are between 0.0 and 1.0

Reply:
The FFTTap sometimes returns data that is .NaN - this would cause an exception to occur (not sure what the origin of this is, but I am sure this is why this code was added). Does vDSP_vclip resolve NaN as zero?

https://developer.apple.com/documentation/swift/floatingpoint/1641763-isnan

Reply:
Good call. I assumed that vDSP_vclip handles NaNs, but i just checked and it does not. I will fix.

Reply:
Added handling of NaN's here: https://github.com/AudioKit/AudioKitUI/pull/25/commits/407132fd704ae3b6d44836dba427827ed4e424da

I opted to do it when producing the final numbers to protect against any of the other calculations producing NaNs.